### PR TITLE
executor: delete the obsolete Next() function in TableReaderExecutor

### PR DESCRIPTION
**Pull request by wsabc01**
delete executor/distsql.go: TableReaderExecutor

### CODE REVIEW

File: executor/distsql.go [hunk 1]
```diff
@@ -219,48 +219,6 @@ func (e *TableReaderExecutor) Close() error {
...
 
-// Next returns next available Row. In its process, any error will be returned.
-// It first try to fetch a partial result if current partial result is nil.
-// If any error appears during this stage, it simply return a error to its caller.
-// If it successfully initialize its partial result, it will use this to get next
-// available row.
-func (e *TableReaderExecutor) Next(ctx context.Context) (Row, error) {
-	for {
-		// Get partial result.
-		if e.partialResult == nil {
```
Comment (winkyao) [inline executor/distsql.go]: Please remove TableReaderExecutor.partialResult field. It's useless.

**Comment by XuHuaiyu** [comment]
LGTM
PTAL @winkyao

**Pull request merged**